monokai-theme.el

```diff
@@ -430,9 +430,9 @@ Also affects 'linum-mode' background."
 
    `(highlight
      ((,monokai-class (:background ,monokai-highlight
-                                   :distant-foreground nil))
+                                   :distant-foreground 'unspecified))
       (,monokai-256-class (:background ,monokai-256-highlight
-                                       :distant-foreground nil))))
+                                       :distant-foreground 'unspecified))))
```

Comment:
You need to use `unspecified`, not `'unspecified` (delete the quote).

I just tested it and with the quote you get:
```
 ■  Error (use-package): monokai-theme/:config: Wrong type argument: stringp, 'unspecified
```

Reply:
implemented, thank you for catching this!